opentelemetry-sdk/src/opentelemetry/sdk/_metrics/export/__init__.py

```diff
@@ -12,14 +12,27 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+import logging
+import os
+import threading
```

Comment:
Nit: using `from X import Y` is consistent with the rest of imports and leads to shorter code :+1: